images/base/Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN mamba install --yes  -c conda-forge -c blsqtech  \
   'gcsfs>=2025,<2026' \
   'papermill>=2.6,<2.7' \
   'psycopg2-binary=2.*' \
+  'duckdb>=1.3,<1.4' \
```

Comment:
it's perhaps better duckdb>=1.3,<1.5 (cfr my comment under the PR)

Reply:
Done! Good catch on the LTS version.